controllers/storagecluster/cephrbdmirrors.go

```diff
@@ -124,3 +130,283 @@ func (obj *ocsCephRbdMirrors) ensureDeleted(r *StorageClusterReconciler, sc *ocs
 	}
 	return reconcile.Result{}, r.deleteCephRbdMirrorInstance(cephRbdMirrors)
 }
+
+func (r *StorageClusterReconciler) ensureRbdMirrorDebug(sc *ocsv1.StorageCluster) error {
+	job := &batchv1.Job{}
+	err := r.Client.Get(context.TODO(), types.NamespacedName{
+		Name:      "rbd-mirror-debug",
+		Namespace: sc.Namespace,
+	}, job)
+
+	if sc.Spec.Mirroring.Enabled {
+		if err != nil {
+			if errors.IsNotFound(err) {
+				// Create the debug job
+				return r.Client.Create(context.TODO(), rbdMirrorDebugJob(sc))
+			}
+			return err
+		}
+		return nil
+	} else if err == nil {
+		// mirroring is not enabled but the job exists
+		err := r.Client.Delete(context.TODO(), job)
+		if err != nil {
+			return err
+		}
+		// Create a undo job to remove the ceph configs applied by the debug job
+		err = r.Client.Create(context.TODO(), undoRbdMirrorDebugJob(sc))
+		if err != nil {
```

Comment:
you also need to take care of the already existing case here

```suggestion
if err != nil &&  errors.IsAlreadyExist(err){
	return err
}
```